Adyen/Core/Components/Base/BasicComponentConfiguration.swift

```diff
@@ -6,19 +6,19 @@
 
 import Foundation
 
-/// :nodoc:
+/// Any component's most basic configuration.
 public protocol AnyBasicComponentConfiguration: Localizable {
     
     var style: FormComponentStyle { get }
 }
 
-/// :nodoc:
+/// The configuration of any component thats aware of shoppers' personal information.
 public protocol AnyPersonalInformationConfiguration: AnyBasicComponentConfiguration {
     
     var shopperInformation: PrefilledShopperInformation? { get }
 }
 
-/// :nodoc:
+/// Any component's most basic configuration.
 public struct BasicComponentConfiguration: AnyBasicComponentConfiguration {
```

Comment:
just a thought:

`BasicComponentConfiguration` should be protocol, and `AnyBasicComponentConfiguration` - struct
Same for `PersonalInformationConfiguration ` and `AnyPersonalInformationConfiguration`